src/PresentationalComponents/Notification/Notification.js

```diff
@@ -32,16 +31,19 @@ import FormButtons from '../shared/FormButtons';
 import FormRender from '@data-driven-forms/react-form-renderer/form-renderer';
 import { useDispatch, useSelector } from 'react-redux';
 import { getNotificationSchema, saveNotificationValues } from '../../actions';
+import { notificationConfigForBundle } from '../../Utilities/functions';
 import { notificationPreferences, register } from '../../store';
 import unsubscribe from '../../config/data/unsubscribe.json';
 import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/redux';
+import getBaseName from '../../Utilities/getBaseName';
 
 const Notification = () => {
   const { bundleName } = useParams();
   const dispatch = useDispatch();
   const store = useSelector(
     ({ notificationPreferences }) => notificationPreferences
   );
+  const bundleDisplayTitle = notificationConfigForBundle(bundleName).title;
```

Comment:
You should use safe accessor here
```suggestion
  const bundleDisplayTitle = notificationConfigForBundle(bundleName)?.title;
```